lib/services/local/session.go

```diff
@@ -95,3 +98,249 @@ func (s *IdentityService) DeleteAllAppSessions(ctx context.Context) error {
 	}
 	return nil
 }
+
+// GetWebSession returns a web session state for the given user and session id
+func (s *IdentityService) GetWebSession(user, sessionID string) (types.WebSession, error) {
+	return s.WebSessions().Get(context.TODO(), proto.GetWebSessionRequest{
+		User:      user,
+		SessionID: sessionID,
+	})
+}
+
+// UpsertWebSession updates or inserts a web session for the user and the session ID
+func (s *IdentityService) UpsertWebSession(user, sessionID string, session types.WebSession) error {
+	session.SetUser(user)
+	session.SetName(sessionID)
```

Comment:
shouldn't the caller populate the session fields?

Reply:
This is from my attempt to keep the old APIs for whatever reason. I now believe I can remove these as well.